packages/neos-ui-editors/src/Editors/Reference/index.js

```diff
@@ -48,6 +69,7 @@ export default class ReferenceEditor extends PureComponent {
             options={sanitizeOptions(options)}
             value={value}
             onValueChange={this.handleValueChange}
+            onClick={() => this.handleClick(options[0].uri)}
```

Comment:
This works :) 

When I tested this in combination with #3341, I drilled through the `<SelecBox/>` call chain and noticed, that `onClick` is missing in the `propTypes` declarations of the following components:

- `SelectBox`
- `SelectBox_Header`

I'd like to suggest however, that `SelectBox` should accept `onHeaderClick` and forward it as `onClick` to the `SelectBox_Header`. This way, it's a clearer API.